utils/vscode/package.json

```diff
@@ -33,16 +36,22 @@
         "scopeName": "source.carbon",
         "path": "./carbon.tmLanguage.json"
       }
+    ],
+    "snippets": [
+      {
+        "language": "carbon",
+        "path": "./carbon.code-snippets"
+      }
     ]
   },
+  "keywords": [
+    "carbon",
+    "snippets"
+  ],
   "scripts": {
     "package": "mkdir -p out && vsce package -o out/carbon.vsix"
   },
-  "devDependencies": {
-    "@vscode/vsce": "^2.27.0"
```

Comment:
Why drop this dependency? Are you sure it's not needed?

Reply:
I will fix in next commit